app/exerlog/lib/UI/prev_workout/delete_prev_workout_dialog.dart

```diff
@@ -4,7 +4,8 @@ import 'package:flutter/material.dart';
 
 class DeleteWorkoutAlert extends StatefulWidget {
   RaisedGradientButton deleteWorkoutButton;
-  DeleteWorkoutAlert(this.deleteWorkoutButton);
+  RaisedGradientButton cancelWorkoutButton;
```

Comment:
I know this PR is just about adding the cancel button, so this might be a separate ticket, but we probably shouldn't be sending the entire button widget to this alert. Ideally we should just pass 2 string values for the accept and decline button. This way the styling of the alert stays in the alert file. We can also return a boolean or maybe an enum to specify which button the use clicked (accept or decline) and then handle the logic from the file we called the alert in, for example.

```
 final result = showAlert();
 if(result == AlertResult.decline) return;
 DeleteWorkout()
```

Reply:
Yeah I agree, but maybe like you said this could be a separate issue? Because I think there are quite a few things that need to be rewritten (not a ton but still a bit of work) for this to work.

Reply:
@KalleHallden should i rewrite the whole thing as one button and alert?